STULabel.xcodeproj/xcshareddata/xcschemes/Demo.xcscheme

```diff
@@ -53,6 +51,7 @@
       debugXPCServices = "NO"
       stopOnEveryThreadSanitizerIssue = "YES"
       stopOnEveryUBSanitizerIssue = "YES"
+      migratedStopOnEveryIssue = "YES"
```

Comment:
I'm not 100% sure this `xcscheme` update is necessary or not, but the migrator of the Xcode has done this. I think this is not harmful so it can be left as-is.